etc/onespan-sign-node.api.md

```diff
@@ -159,7 +266,7 @@ export interface DocumentApproval {
     // (undocumented)
     accepted: Nullable<string>;
     // (undocumented)
-    data: Nullable<Record<string, any>>;
+    data: CustomData;
```

Comment:
nit – for `Nullable` values, I would prefer to keep `Nullable` out of types.

```ts
// Current
type CustomData = Nullable<Record<string, any>>;
data: CustomData

// My preference
type CustomData = Record<string, any>;
data: Nullable<CustomData>
```

I think it's a bit easier to understand just by glancing at it versus having to go to the type definition of `CustomData`.